.pylintrc

```diff
@@ -23,7 +23,20 @@ disable=
  old-style-class,
 # "R" Refactor recommendations
  too-few-public-methods,
+# we need, some time, to disable pylint locally :)
+ locally-disabled,
 
 [REPORTS]
 
 msg-template='[{msg_id} {symbol}] {msg} File: {path}, line {line}, in {obj}'
+
+[FORMAT]
+
+# Maximum number of characters on a single line.
+max-line-length=120
```

Comment:
Again, don't add it when it's already there. As I wrote in https://github.com/oamg/convert2rhel/pull/59#discussion_r431192097.

Reply:
Apparently it was [accidentally removed](https://github.com/oamg/convert2rhel/commit/963215b823b872304ec9326972c4d5d45a887751)

Reply:
Aha, I'm sorry then. Thanks @SpyTec.